fmralign/_utils.py

```diff
@@ -49,85 +26,62 @@ def piecewise_transform(labels, estimators, X):
 
     X_transform = np.zeros_like(X)
     for i in np.unique(labels):
-        X_transform[labels == i] = estimators[i].transform(X[labels == i].T).T
+        X_transform[:, labels == i] = estimators[i - 1].transform(
```

Comment:
why `estimators[i-1]` ?

Reply:
because Parcellation class give labels from 1 to n and not from 0 to n-1 (which is more logical)

Reply:
Please add a comment